src_c/constants.c

```diff
@@ -27,6 +27,7 @@
 #include "scrap.h"
 
 /* macros used to create each constant */
+
```

Comment:
I'd like you to delete this added line so this file doesn't become part of the diff.